submission/submission-webapp/src/main/kotlin/ac/uk/ebi/biostd/submission/domain/service/DynamicExtSubmissionSubmitter.kt

```diff
@@ -0,0 +1,46 @@
+package ac.uk.ebi.biostd.submission.domain.service
+
+import ac.uk.ebi.biostd.common.properties.TaskHostProperties
+import ac.uk.ebi.biostd.submission.domain.extended.ExtSubmissionSubmitter
+import ac.uk.ebi.biostd.submission.domain.extended.LocalExtSubmissionSubmitter
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.withContext
+import mu.KotlinLogging
+import java.io.File
+import java.util.UUID
+
+private val logger = KotlinLogging.logger {}
+
+class DynamicExtSubmissionSubmitter(
```

Comment:
Why isn't this implementation located in the same place as the Local one? I think both are concerns of the submission core, not the web layer